app/views/application_records/show.haml

```diff
@@ -65,6 +69,40 @@ by:
                          required: true,
                          value: @record.position.default_interview_location
       .note_item= submit_tag 'Review application and schedule interview'
+    - if !@record.saved_for_later
```

Comment:
You have "if not X" followed by "if X". Seems like "if X" and "else", with the respective code reversed, would be simpler.